frame/contracts/src/wasm/mod.rs

```diff
@@ -1555,8 +1555,8 @@ mod tests {
 
 		let gas_left = Weight::decode(&mut &*output.data).unwrap();
 		let actual_left = ext.gas_meter.gas_left();
-		assert!(gas_left < gas_limit, "gas_left must be less than initial");
-		assert!(gas_left > actual_left, "gas_left must be greater than final");
+		assert!(gas_left.any_lt(gas_limit), "gas_left must be less than initial");
+		assert!(gas_left.any_gt(actual_left), "gas_left must be greater than final");
```

Comment:
I think just calling a contract should reduce any resource.

```suggestion
		assert!(gas_left.all_lt(gas_limit), "gas_left must be less than initial");
		assert!(gas_left.all_gt(actual_left), "gas_left must be greater than final");
```